packages/components/src/components/contract-card/result-overlay/result-overlay.tsx

```diff
@@ -30,17 +31,10 @@ export const ResultStatusIcon = ({ getCardLabels, is_contract_won }: TResultStat
             'dc-result__caption--lost': !is_contract_won,
         })}
     >
-        {is_contract_won ? (
-            <React.Fragment>
-                {getCardLabels().WON}
-                <Icon icon='IcCheckmarkCircle' className='dc-result__icon' color='green' />
-            </React.Fragment>
-        ) : (
-            <React.Fragment>
-                {getCardLabels().LOST}
-                <Icon icon='IcCrossCircle' className='dc-result__icon' color='red' />
-            </React.Fragment>
-        )}
+        <React.Fragment>
```

Comment:
Do you need this `React.Fragment` here? Seems that it's inside `<span/>`

Reply:
yes. React.Fragment needs to wrap Icon and Span into a single element

Reply:
oh whooops lol